History.md

```diff
@@ -1,5 +1,10 @@
 ## v.NEXT
 
+* `observe`/`observeChanges` callbacks are now bound.
+  The same `EnvironmentVariable`s that was present when
+  `observe`/`observeChanges` was called is now available inside the callbacks.
```

Comment:
I think we should explain what binding environment is in documentation (see https://github.com/meteor/docs/issues/139) and then here just say:

>  `observe`/`observeChanges` callbacks have now their Meteor environment bound.

With "Meteor environment bound" linking to description of `Meteor.bindEnvironment`.